lib/Applify.pm

```diff
@@ -330,7 +359,7 @@ sub _calculate_option_spec {
     $option->{default}
       and ref $option->{default} ne 'ARRAY'
       and die 'Usage option ... default => [Need to be an array ref]';
-    $option->{default} ||= [];
+    $option->{default} = [];
```

Comment:
[this test](https://github.com/kiwiroy/applify/blob/mojo-file/t/file-dir.t#L65) fails using conditional assignment `||=`. No other tests fail when unconditionally assigning. That's not to say unconditionally assigning is correct, just that this functionality is uncovered in tests. Still investigating...